packages/tracking/src/Tracking.php

```diff
@@ -5,26 +5,14 @@
 
 namespace Automattic\Jetpack;
 
-require_once dirname( dirname( dirname( __DIR__ ) ) ) . '/_inc/lib/tracks/client.php';
-
 class Tracking {
-	static $product_name = 'jetpack';
-
-	static function track_jetpack_usage() {
-		if ( ! \Jetpack::jetpack_tos_agreed() ) {
-			return;
-		}
+	public $product_name;
 
-		// For tracking stuff via js/ajax
-		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'enqueue_tracks_scripts' ) );
-
-		add_action( 'jetpack_activate_module', array( __CLASS__, 'track_activate_module' ), 1, 1 );
-		add_action( 'jetpack_deactivate_module', array( __CLASS__, 'track_deactivate_module' ), 1, 1 );
-		add_action( 'jetpack_user_authorized', array( __CLASS__, 'track_user_linked' ) );
-		add_action( 'wp_login_failed', array( __CLASS__, 'track_failed_login_attempts' ) );
+	function __construct( $product_name = 'jetpack' ) {
```

Comment:
It doesn't seem like we need to change the product name. YAGNI?

Reply:
I imagine `VaultPress` passing `vaultpress`?

Reply:
👍 Let's keep it then